apps/desktop/desktop/src/main/java/bisq/desktop/components/controls/SelectableLabel.java

```diff
@@ -78,16 +77,31 @@ public SelectableLabel(String text) {
         textArea.setContextMenu(new ContextMenu());
         textArea.setText(text);
         textArea.getStyleClass().addAll("selectable-label", "hide-vertical-scrollbar");
-        label.heightProperty().addListener(labelHeightListener);
+        label.boundsInParentProperty().addListener(labelBoundsListener);
         getStyleClass().addListener(new WeakListChangeListener<>(styleClassListener));
     }
 
+    public void setEditable(boolean editable){
+        this.textArea.setEditable(editable);
+    }
+
+    public String getText() {
+        return this.textArea.getText();
+    }
+
     private void swap() {
-        label.heightProperty().removeListener(labelHeightListener);
-        textArea.setMinWidth(label.getWidth());
-        textArea.setMaxWidth(textArea.getMinWidth());
-        textArea.setMinHeight(label.getHeight());
-        textArea.setMaxHeight(textArea.getMinHeight());
+        if (isInSelectionMode) {
+            return;
+        }
+        // double-check dimensions here, just in case runLater was called prematurely
```

Comment:
"runLater was called prematurely" - i dont think that can be the case. Only reasons I can imagine is that bounds are not 100% aligend with label.width/height or that in the delay the w/h get reset to 0 (dont expect). I think both cases are close to 0 likely. If you cannot reproduce the issue with out that extra check I would suggest to remove it, as it seems not needed.